SimExLite/SourceCalculators/GaussianSourceCalculator.py

```diff
@@ -0,0 +1,207 @@
+""":module GaussianSourceCalculator: Module that holds the GaussianSourceCalculator class.  """
+
+import sys
+import numpy as np
+from pathlib import Path
+from scipy.constants import hbar, c
+from pint import Quantity, Unit
+import imp
+
+from libpyvinyl import BaseCalculator, CalculatorParameters
+from SimExLite.WavefrontData import WavefrontData
+
+
+class GaussianSourceCalculator(BaseCalculator):
+    """:class GaussianSourceCalculator: Class representing a x-ray free electron laser photon source."""
+
+    def __init__(
+        self,
+        name: str,
+        WPG_path: str,
```

Comment:
remove `WPG_path`: wpg must be installed in the environment or be on $PYTHONPATH.

Reply:
Unfortunately, I didn't find an instruction to install WPG into the environment in their documentation. Also, it's not possible to `export PYTHONPATH=XXX` in a jupyter-hub environment. 

Do you know any good way to install WPG into the environment or set the $PYTHONPATH in a jupyter-hub environment?

Reply:
I'd write somewhere in the documentation:
SimEx-Lite depends on WPG (github.com/Samoyl/WPG). Please download and build WPG as per their documentation. Then, you have to make the WPG library "visible" to SimEx-Lite. To this end, do one of the following:
1. `$PYTHONPATH`: If you have the environment variable `$PYTHONPATH` set, copy the WPG/ directory into any directory listed in `$PYTHONPATH`.
2. `conda`: If you installed SimEx-Lite in a conda environment, activate that environment and copy the WPG/ directory to $CONDA_PREFIX/lib/python3.x/site-packages/ (x is your python minor version).
3. `pip`: If you installed SimEx-Lite with `pip install`, copy the WPG directory into $HOME/.local/lib/python3.x/site-packages

Test that WPG can be imported by running:
```console
$> python -c 'import WPG.Beamline'
```

Reply:
Needless to say, these instructions should be tested and corrected if needed

Reply:
This problem will be fixed in this issue: #15

Reply:
This problem will be fixed in this issue: #15